content/quickstart/node/tracing.md

```diff
@@ -633,7 +633,7 @@ const defaultConfig = {
   samplingRate: 1.0  // always sample
 };
 
-const exporter = new stackdriver.StackdriverTraceExporter({projectId: 'test-210722'});
+const exporter = new stackdriver.StackdriverTraceExporter({projectId: 'your-project-id'});
```

Comment:
In other languages we've used `GCP_PROJECTID` instead of a string.

In Node.js, you can retrieve from the environment using `process.env.GCP_PROJECTID`, please use that.

Reply:
`process.env.GCP_PROJECTID` returned null, this is how I ended up retrieving the id:

```js
function getProjectId() {
  return require(process.env.GOOGLE_APPLICATION_CREDENTIALS).project_id;
}

const exporter = new stackdriver.StackdriverTraceExporter({projectId: getProjectId()});
```
Let me know if my implementation isn't what you meant! Thanks again for reviewing.

Reply:
Gotcha, thanks! That can work but in Javascript you can also do `const projectId = process.env.GCP_PROJECTID || 'census-demos'` as the other demos do. However, let's merge this in, yours is perfect as is, thank you!